public/game.js

```
@@ -0,0 +1,1286 @@
+      oscillator.frequency.setValueAtTime(659, audioContext.currentTime + 0.1);
+      oscillator.frequency.setValueAtTime(784, audioContext.currentTime + 0.2);
+      gainNode.gain.value = 0.15;
+      oscillator.start();
+      oscillator.stop(audioContext.currentTime + 0.3);
+      break;
+
+    case 'levelup':
+    case 'fanfare':
+      // Arpeggio
+      const notes = [523, 659, 784, 1047];
+      notes.forEach((freq, i) => {
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)